Makefile

```diff
@@ -18,18 +22,60 @@ clean: ## delete most git-ignored files
 	find . -name '*.pyc' -exec rm -f {} +
 	find . -name '*.pyo' -exec rm -f {} +
 	find . -name '*~' -exec rm -f {} +
+	coverage erase
+	rm -fr *.egg-info
 
 requirements: ## install environment requirements
 	pip install -r requirements/base.txt
 
+install-automation-reqs: ## install tox
+	$(TOX) pip install -r requirements/tox.txt
+
+tox-test-reqs: install-automation-reqs
+	$(TOX) pip install -r requirements/test.txt --exists-action w
+
 upgrade: export CUSTOM_COMPILE_COMMAND=make upgrade
 upgrade: ## update the requirements/*.txt files with the latest packages satisfying requirements/*.in
 	pip install -qr requirements/pip-tools.txt
 	# Make sure to compile files after any other files they include!
 	$(PIP_COMPILE) -o requirements/pip-tools.txt requirements/pip-tools.in
 	$(PIP_COMPILE) -o requirements/base.txt requirements/base.in
 	$(PIP_COMPILE) -o requirements/test.txt requirements/test.in
+	$(PIP_COMPILE) -o requirements/tox.txt requirements/tox.in
+
+quality: clean tox-test-reqs## check coding style with pycodestyle and pylint
```

Comment:
if I run this in a local machine why should I need to install tox requirements ? btw this fail in a new environment due to the lack of coverage requirement

Reply:
[77e195d](https://github.com/eduNEXT/eox-nelp/pull/19/commits/77e195d4e6396ee07759da4902b0e6b90980dbcd)